pkg/agent/openflow/pipeline.go

```diff
@@ -348,6 +356,9 @@ var (
 	// a SNAT IP. The bit range must match SNATIPMarkMask.
 	snatPktMarkRange = binding.Range{0, 7}
 
+	// serviceTrafficSrcMAC takes 0..47 range of ct_label to store the source MAC address of Service traffic.
+	serviceTrafficSrcMacRange = binding.Range{0, 47}
```

Comment:
nodePortTrafficSrcMACRange to be more specific as Service includes clusterIP?

Reply:
Yes. In the future, this will be used for rewriting destination MAC address for all Service whose request is from remote hosts.